ALCF/helpers.sh

```diff
@@ -557,21 +548,27 @@ make_ds_hostfile() {
 }
 
 ezpz_savejobenv() {
-    local outfile="${WORKING_DIR}/bin/savejobenv" ; mkdir -p $(dirname "${outfile}")
-    curl -Ls https://raw.githubusercontent.com/saforem2/ezpz/main/src/ezpz/bin/savejobenv > "${outfile}" && source "${outfile}"
+    file=$(mktemp)
+    curl -Ls https://raw.githubusercontent.com/saforem2/ezpz/main/src/ezpz/bin/savejobenv > "${file}"
+    source "${file}" || exit
 }
 
 ezpz_getjobenv() {
-    local outfile="${WORKING_DIR}/bin/getjobenv" ; mkdir -p $(dirname "${outfile}")
-    curl -Ls https://raw.githubusercontent.com/saforem2/ezpz/main/src/ezpz/bin/getjobenv > "${outfile}" && source "${outfile}"
+    file=$(mktemp)
+    curl -Ls https://raw.githubusercontent.com/saforem2/ezpz/main/src/ezpz/bin/getjobenv > "${file}"
+    source "${file}" || exit
```

Comment:
Similarly to `ezpz_savejobenv`, use `mktemp` for temporary file creation, download `ezpz/src/ezpz/bin/getjobenv`, and then source it.